### DISCUSSION TITLE
articles/buying-property

### DISCUSSION THREAD

**Discussion by hutusi** [hutusi/hutusi.github.com#149] [Comments]
http://hutusi.com/articles/buying-property

   当你想要回家，却找不到回家的路 当你一路南下，却找不到一件差事 你就决定去芝加哥  哭泣和祈祷都帮不了你  —— Led Zeppelin 《When the levee breaks》

**Comment by fengguangle** [comment]
太难了，一个90后沪漂，偶然看到你这个文章。看到最后一段话的时候，我竟然潸然泪下！
年

**Comment by hutusi** [comment]
@fengguangle 是的，沪漂很难，但年轻就是资本，加油✊

**Comment by yuesh1** [comment]
如此生活三十年
....................

**Reply by hutusi** [reply]
快四十年了😅

**Comment by mojiaorg** [comment]
但实际上相对于大多数90后，00后而言，你是幸运的，你在11年已经上车了。
我在北京，2016年才毕业工作，眼看这着2016年前后房价翻番，却毫无还手之力。
但没办法，人总是要生活，房子对于我而言是刚需，就在去年，我们终于买房了，但与2016年相比，涨了一倍多。
在这短短的6年多出的钱是时代给我们后来者无形中增加的石头。但我却毫无办法。
时代的一粒沙，在一个人身上就是一座山。

**Reply by hutusi** [reply]
确实，70后相对80后幸运，80后相对90后幸运些。00后是不是烦恼少些呢~

**Reply by wangyu-xreal** [reply]
北京16年那波涨的很厉害，最近跌了点，还是比15年高很多。
有个同学在房山买的房，曾说那几年涨幅顶他好几年工资。买的晚的就只能背着更多的房贷。
15年某小区单价3.9w左右， 16年飙到6w，  最近降到5w左右了。

其实除去首付，同一套房的房贷也比房租贵很多，但是房产证绑定太多东西，不得不买房。

**Reply by hutusi** [reply]
是啊，对大多数人来说，买房是个躲不过的话题。

**Reply by Gimorocun** [reply]
00后初到上海工作😅，现在觉得租房子自己住的舒心就很满足了，买房想都不敢想，何况房产又是下行不知道会持续多少年

看了博主几篇文章，文笔太好了，最近也打算开始写博客，希望10年后能达到博主的文学水平🥰

**Reply by hutusi** [reply]
@Gimorocun 谢谢，一起加油💪🏻
